pinot-query-runtime/src/test/resources/queries/TableExpressions.json

```diff
@@ -16,6 +16,9 @@
       }
     },
     "queries": [
+      { "sql": "SELECT * FROM {tbl} WHERE strCol IN (SELECT strCol FROM {tbl} WHERE intCol < 100)" },
```

Comment:
may be consider adding a test for float / double as well and test for both `zero point something` and `nonzero point something` if not already covered ?

Reply:
actually IN cannot produce a double or a float. this will be address separately